docs/types/value-types.rst

```diff
@@ -182,7 +181,7 @@ Operators:
 Address
 -------
 
-The address type comes in two flavours, which are largely identical:
+The address type comes in two largely identical flavors:
```

Comment:
Also, we generally use British spelling :) Though I usually forget about it myself - I'm just too used to the US one.
```suggestion
The address type comes in two largely identical flavours:
```

Reply:
We do??? I've read in multiple places words written in international English (US).

Reply:
I mean, we have a rule. Whether we're applying it consistently is another matter.

I often catch myself not remembering about it myself, which is why I approved the PR anyway :)

Reply:
Well, I kind of challenge that "rule". British is not International English. 🔥

Reply:
I would not be against not using British English :) But the law requires me to point this out as a typo :P